contents/docs/deployment/deploy-proxy.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Deploying a Reverse Proxy to Posthog Cloud 
+sidebar: Docs
+showTitle: true
+---
+<br>
+
+
+## Why host this reverse proxy? 
+
+You'll want to host this reverse proxy if you want to use Posthog Cloud but still want tracking data to be sent first to your domain, then to us. This helps with tracker blockers or if you have migrated or are migrating off Posthog open source hosted locally to Posthog Cloud.
+
+## Deploying the reverse proxy
+
+We like using Caddy because it makes setting up the reverse proxy and TLS a breeze.
+
+```bash
+docker run -p 80:80 -p 443:443 caddy caddy reverse-proxy --to p.posthog.com:443 --from <YOUR_TRACKING_DOMAIN>
+```
+
+You'll want to sub out `YOUR_TRACKING_DOMAIN` for whatever domain you use for proxying to Posthog. I'd suggest something like `e.yourdomain.com` or the like.
```

Comment:
Typo: PostHog

Reply:
Just chiming in to nitpick that in docs "we" is preferred to "I" :D

Reply:
👍  good catch!